modules/zetaBidAdapter.md

```diff
@@ -0,0 +1,39 @@
+# Overview
+
+```
+Module Name: Zeta Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: DL-ZetaDSP-Supply-Engineering@zetaglobal.com
+```
+
+# Description
+
+Module that connects to Zeta's demand sources
+
+# Test Parameters
+```
+        var adUnits = [
+            {
+                mediaTypes: {
+                    banner: {
+                        sizes: [[300, 250]],  // a display size
+                    }
+                },
+                bids: [
+                    {
+                        bidder: 'Zeta Global',
```

Comment:
The server request failed when I submitted these test bids.

Reply:
Both issues should be addressed